### ISSUE TITLE
Microsoft Search Datasource error: Token request previously failed

### ISSUE THREAD

**Issue by yvsuhas** [microsoft-search/pnp-modern-search#2374]
**Version Used**
4.7

**Describe the bug**
When using the PnP Search Results Webpart and selecting Microsoft Search as the data source after granting the necessary graph api permissions, the webpart by default shows "Token request previously failed" error with 0 results.

**To Reproduce**
1. Go to a SharePoint Online page
2. Add a PnP Search Results Webpart
3. Edit the webpart and select "Microsoft Search" as a data source.
4. Select other items from the dropdown "Entity types to search" such as External Items, List Items, List, Sites.
5. 0 results with error: Token request previously failed appears

**Expected behavior**
Search results from the site appears that come from OneDrive, SharePoint.

**Screenshots**
![image](https://user-images.githubusercontent.com/19211871/191380862-790b3b71-18be-4fd7-b506-2a0cf943d739.png)

**Desktop (please complete the following information):**
 - Browser: Chrome, Edge

**Additional context**
I've upgraded from 4.5 to 4.7 to see if it fixes the issue and when I've uploaded the new package, it didn't ask me to grant the api permissions again in the SharePoint admin center.

**Comment by wobba** [comment]
Can you take a look at the network trace or console log in dev tools in your browser? See if you find anything auth issue related? Also try to clear all cache or use a private browser window, possible another user as well.

**Comment by FranckyC** [comment]
You can also try with the latest 4.8.0-beta version https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.8.0

**Comment by yvsuhas** [comment]
Thank you for your comments. I've tried Wobba's suggestions and got nothing out of the console.log except the error itself and nothing in the network trace. I've checked the Azure AD App registrations to make sure that the required permissions are granted to the graph service. Console message is Uncaught (Promise) Status Code -1 and requestId is null. I will give 4.8 beta a try as well but my upgrade from 4.5 to 4.7 didn't help in this case.
